docs/source/examples/02_advanced_fmnist.rst

```diff
@@ -5,20 +5,47 @@ Advanced Example
 In this second example, we will take a look into a couple more advanced features
 and ways to customize **Cockpit**.
 
+Just like before, you also need the :download:`utility file 
+<../../../examples/_utils_examples.py>` 
+which provides us with the training data, a convolutional network and a logpath.
+You can copy all `example files 
+<https://github.com/f-dangel/cockpit/tree/development/examples>`_ from our 
+repository.
+
+.. literalinclude:: ../../../examples/02_advanced_fmnist.py
+   :language: python
+   :linenos:
+   :emphasize-lines: 12,19-28,44,60-67,72-73,75-82
+
 To run this example, run 
 
 .. code:: bash
 
   python 02_advanced_fmnist.py
 
-Assuming you have a similar structure to our `example files 
-<https://github.com/f-dangel/cockpit/tree/development/examples>`_
-from the repository, this will execute the following file:
-
-.. literalinclude:: ../../../examples/02_advanced_fmnist.py
-   :language: python
-   :linenos:
-   :emphasize-lines: 12,19-27,43,59-65,70-71,73-80
+This time no **Cockpit**-plot will show. Instead, the plots will be directly saved
+to files. Everything that the **Cockpit** tracked during training will also be stored
+and both this *logfile* as well as the plots can be inspected and analyzed after 
+training is finished.
+
+.. code-block:: console
+
+  $ python 02_advanced_fmnist.py
+  
+  Step:     1 | Loss: 2.2965
+  Step:     2 | Loss: 2.3185
+  Step:     3 | Loss: 2.2932
+  Step:     4 | Loss: 2.2893
+  Step:     5 | Loss: 2.2865
+  Step:     6 | Loss: 2.2780
+  Step:     7 | Loss: 2.2485
+  Step:     8 | Loss: 2.2600
+  Step:     9 | Loss: 2.1823
+  Step:    10 | Loss: 2.0557
+  [cockpit|plot] Saving figure in /home/frank/PhD/Nextcloud/PhD/Projects/2019-10-Cockpit/cockpit/logfiles/cockpit_output__primary10.png
```

Comment:
@fsschneider Maybe you want to replace `/home/frank/PhD/Nextcloud/PhD/Projects/2019-10-Cockpit/` with `~/`